server/src/domain/media/media.service.ts

```diff
@@ -228,7 +236,15 @@ export class MediaService {
     }
   }
 
-  private getCodecConfig(config: SystemConfigFFmpegDto) {
+  async getCodecConfig(config: SystemConfigFFmpegDto) {
+    if (!config.accel || config.accel === TranscodeHWAccel.DISABLED) {
```

Comment:
Did we decide this would always have a value?

Reply:
This is from back when I wasn't sure if it would have a value, but yeah it always should